### PR TITLE
6038:  CR schema creation refactoring - unify the SQL 

### PR DESCRIPTION
**Pull request by raman-bt**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/centralrepository/optionspanel/EamDbSettingsDialog.java [hunk 1]
```diff
@@ -447,12 +448,19 @@ private void createDb() {
...
+                        result = centralRepoSchemaFactory.initializeDatabaseSchema()
+                            && centralRepoSchemaFactory.insertDefaultDatabaseContent();
+                    } catch (CentralRepoException ex) {
+                       logger.log( Level.SEVERE, "Unable to initialize database schema or insert contents into Postgres central repository.", ex);
+                    }
                 }
                 if (!result) {
                     // Remove the incomplete database
                     if (dbCreated) {
+                        // RAMAN TBD: migrate  deleteDatabase() to RdbmsCentralRepoSchemaFactory
```
[the remark this answers is not in the document]
Comment (raman-bt) [reply Core/src/org/sleuthkit/autopsy/centralrepository/optionspanel/EamDbSettingsDialog.java]: Not for this PR

**Comment by APriestman** [comment]
The codacy issues are not valid

**Comment by raman-bt** [comment]
> Please address the Codacy issues. Also, I don't think calling the new class a "schema factory" is quite accurate. It is responsible for creating databases, not database schemas.

Codacy is complaining about making sure that the connection is closed, which it is - in a finally block. I'm nor sure why Codacy isn't detecting it or what it's really complaining about.

As for name of the class - I think what we care creating is a database structure - i.e. tables/indexes ( a schema in my understanding)  The 'database' itself is a just a named entity - in this case 'central_repository'. That 'database' is created in the settings class(es).

**Comment by rcordovano** [comment]
Moving the discussion of my comment about the class name into Jira...

**Pull request merged**